### PR TITLE
'synchoronizing' word error

### PR DESCRIPTION
**Pull request by StudyNick**
spelling mistakes, 'synchoronizing' modify 'Synchronizing'

### CODE REVIEW

**Comment by StudyNick** [comment]
I signed it!

File: pkg/controller/service/servicecontroller.go [hunk 1]
```diff
@@ -47,9 +47,9 @@ import (
 )
 
 const (
-	// Interval of synchoronizing service status from apiserver
+	// Interval of Synchronizing service status from apiserver
```
Comment (gyliu513) [inline pkg/controller/service/servicecontroller.go]: s/Synchronizing/synchronizing

File: pkg/controller/service/servicecontroller.go [hunk 2]
```diff
@@ -47,9 +47,9 @@ import (
...
 	serviceSyncPeriod = 30 * time.Second
-	// Interval of synchoronizing node status from apiserver
+	// Interval of Synchronizing node status from apiserver
```
Comment (gyliu513) [inline pkg/controller/service/servicecontroller.go]: ditto

Comment (StudyNick) [inline pkg/controller/service/servicecontroller.go]: 'synchoronizing' ,chor is error, not 'o'

File: pkg/controller/service/servicecontroller.go (hunk 1 above)
Comment (MrHohn) [inline pkg/controller/service/servicecontroller.go]: Yeah, please make it lowercase.

File: pkg/controller/service/servicecontroller.go (hunk 2 above)
Comment (gyliu513) [inline pkg/controller/service/servicecontroller.go]: Yes, I mean we need to use lower case here ;)

Comment (StudyNick) [inline pkg/controller/service/servicecontroller.go]: @gyliu513 ok.updated

**Comment by gyliu513** [comment]
lgtm, thanks @StudyNick

**Comment by MrHohn** [comment]
@StudyNick Could you squash the commits? Thanks.

**Comment by StudyNick** [comment]
@MrHohn ok,finish!

**Comment by MrHohn** [comment]
I think you squashed incorrectly, you are now including an commit from another people.

**Comment by StudyNick** [comment]
@MrHohn sorry, i used 'git rebase -i HEAD~2',but I didn't see the other commit.
 What should I do?

**Comment by MrHohn** [comment]
One way I could think of is to drop this commit and then re-commit with your own changes.

**Comment by StudyNick** [comment]
@MrHohn ok

**Pull request closed without merging**